fastai/column_data.py

```diff
@@ -137,12 +137,14 @@ def forward(self, x_cat, x_cont):
 class StructuredLearner(Learner):
     def __init__(self, data, models, **kwargs):
         super().__init__(data, models, **kwargs)
+
+    def _determine_loss_func(self, data):
         if data.is_reg:
-            self.crit = F.mse_loss
+            return F.mse_loss
```

Comment:
Maybe better to say

    return F.mse_loss if data.is_reg else F.binary_cross_entropy if data.is_multi else F.nll_loss

Reply:
I think readability here is more important than making it short...
>  Readability Counts 

(PEP20)

Also, when and if someone will implement multi-output multi-class (or multi-output regression), things will get even more complicated.

If you're sure about this one liner, tell me. Honestly I don't really see the benefit in this case :)